servers/quarkus-common/src/main/java/org/projectnessie/quarkus/config/CatalogServiceConfig.java

```diff
@@ -22,6 +22,14 @@
 
 @ConfigMapping(prefix = "nessie.catalog.service")
 public interface CatalogServiceConfig {
+  /**
+   * Interval after which a request is retried when storage I/O responds with some "retry later"
+   * response.
+   */
+  @WithName("throttled-retry-after")
```

Comment:
```suggestion
  @WithName("error-handling.throttled-retry-after")
```
So we've got more flexibility to add e.g. exponential backoff/jitter in the future.

Reply:
renamed